README.md

```diff
@@ -170,6 +170,41 @@ const faunadb = require('faunadb')
 + const { logger, showRequestResult } = faunadb
 ```
 
+### Exceptions handling
+
+With the 5.0.0 release, child classes of the base FaunaError class now exist.
+Each class corresponds to an error code from Fauna.
+Inspect the [`errors.js`](https://github.com/fauna/faunadb-js/blob/dcf8c4eece22b344b0a64c76a192634bd747e234/src/errors.js)
+file for more information on how it is implemented.
+
+The following example demonstrates the methods that you can access from the exception object:
+
+```
+import { Client, TransactionAbortedError, AuthenticationFailedError } from 'faunadb'
+
+const client = new Client()
+
+async function run() {
+  try {
+    await client.query()
+  } catch(error) {
+    if(error instanceof TransactionAbortedError) {
+      console.error({
+        code: error.code,
```

Comment:
it's what database returns, like `invalid argument` or  `instance not unique`
httpStatusCode is also available

Reply:
@faunaee I see that @fireridlle has added all the 3 fields:
```
        position: e.position,
        msg: e.message,
        httpStatusCode: e.httpStatusCode
```
and they are the same as in the other drivers

@fireridlle let's remove `code: error.code` since there is no such field in examples for other drivers and we want to keep things consistent as possible